management/status_checks.py

```diff
@@ -675,9 +677,11 @@ def check_mail_domain(domain, env, output):
 	# Stop if the domain is listed in the Spamhaus Domain Block List.
 	# The user might have chosen a domain that was previously in use by a spammer
 	# and will not be able to reliably send mail.
-	dbl = query_dns(domain+'.dbl.spamhaus.org', "A", nxdomain=None)
+	dbl = query_dns(domain+".dbl.spamhaus.org", "A", nxdomain=None)
```

Comment:
Why the change from single to double quotes?

Reply:
the rest of the file is using them, I see I missed the occurrence in 293